### DISCUSSION TITLE
How to change the limit/schemes for settings

### DISCUSSION THREAD

**Discussion by FreshImmuc** [tjarbo/discord-moodle-bot#230] [Q&A]
How do i change the max. length for the Discord Channel ID + how do i change the scheme for the discord username of the admin? In what directory / files are these settings?

**Comment by tjarbo** [comment]
Because of #229: Are u asking for version 1 or 2?

In version 2, the admin's username is validated [here](https://github.com/tjarbo/discord-moodle-bot/blob/abeb2099ab47ef3233757911b665906acc40a7e7/packages/backend/src/controllers/administrator/administrator.schema.ts#L25-L28) and the discord channel id [here](https://github.com/tjarbo/discord-moodle-bot/blob/abeb2099ab47ef3233757911b665906acc40a7e7/packages/backend/src/controllers/connectors/plugins/discordBot.class.ts#L18).

**Reply by FreshImmuc** [reply]
No, i am asking for v1. v2 has someproblems with docker but i already asked that in the other issue and will try to fix this today after school.
